metadata/simpleComputeDataset.json

```diff
@@ -43,12 +39,17 @@
         "allowRawAlgorithm": false,
         "allowNetworkAccess": true,
         "publisherTrustedAlgorithmPublishers": [],
-        "publisherTrustedAlgorithms": []
+        "publisherTrustedAlgorithms": [
```

Comment:
why are we changing  the existing example datasets? In this case even hardcoding the algorithm id?
can we just use the new v5 json for these changes?